src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,74 @@
+import React from 'react';
+
+interface PaginationProps {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const handlePageChange = (page: number) => {
+    let newPage = page;
+
+    if (newPage < 1) {
+      newPage = 1;
+    } else if (newPage > totalPages) {
+      newPage = totalPages;
+    }
+
+    onPageChange(newPage);
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <button
+          data-cy="prevLink"
+          className="page-link"
+          onClick={() => handlePageChange(currentPage - 1)}
+          aria-disabled={currentPage === 1}
+          disabled={currentPage === 1}
+        >
+          «
+        </button>
+      </li>
```

Comment:
Code quality issue: There's a bit of repetition in your JSX, particularly with the `li` and `button` elements. You might want to consider creating a reusable component for a page item. This would make your code DRYer and easier to maintain.